### PR TITLE
Bailey/#15 login controller

### PR DESCRIPTION
**Pull request by BKDuncan**
LoginController finished and data classes updated.

### CODE REVIEW

File: src/business/LoginController.java [hunk 1]
```diff
@@ -1,39 +1,69 @@
...
-     * @return
+     * Attempts to login the user.
+     * @param login The login information being authenticated. In the form: 'username:password'
+     * @return An employee object (with only a username) if the login is valid (Developer or Manager). Null if not valid.
      */
     public Employee validateLogin(String login) {
-        // TODO implement here
+    	// For-each login in the list
+        for(String validLogin : loginInfo_){
+        	if(validLogin.matches("dev-<" + login + ">")){
```
Comment (bill-luu) [inline src/business/LoginController.java:42]: There's no password check here.

File: src/data/Bug.java [hunk 2]
```diff
@@ -6,6 +6,11 @@
 public class Bug {
 
 	/**
+	 * Enum to represent the current bug state
+	 */
+	enum State {pending_approval, rejected, awaiting_assignment, in_progress, fixed};
```
Comment (bill-luu) [inline src/data/Bug.java]: Let's follow conventions on Enum naming, so `PENDING_APPROVAL, REJECTED, etc`
https://stackoverflow.com/questions/3069743/coding-conventions-naming-enums

**Review by bill-luu — CHANGES_REQUESTED** [review]

File: src/business/LoginController.java (hunk 1 above)
Comment (bill-luu) [reply src/business/LoginController.java:42]: Disregard!

Comment (bill-luu) [reply src/business/LoginController.java:42]: We should update the sample in this PR

**Review by bill-luu — CHANGES_REQUESTED** [review]

File: src/data/Bug.java [hunk 3]
```diff
@@ -27,9 +32,9 @@ public Bug() {
 	private int productId_;
 
 	/**
-	 * Add enum here
+	 * Enum for bug state. Can be pending_approval, awaiting_assignment, in_progress, rejected, or fixed.
```
Comment (bill-luu) [inline src/data/Bug.java]: Update this comment too!

**Review by bill-luu — CHANGES_REQUESTED** [review]

**Review by bill-luu — APPROVED** [review]

**Comment by bill-luu** [comment]
LGTM

**Review by a-sood — APPROVED** [review]
Looks good

**Pull request merged**